MessageBird/Net/IRestClient.cs

```diff
@@ -17,9 +17,10 @@ public interface IRestClient
         T Create<T> (T resource) where T : Resource;
         T Retrieve<T>(T resource) where T : Resource;
         void Update(Resource resource);
-        void Delete(Resource resource);
+        T Update<T>(T resource) where T : Resource;
```

Comment:
This does not break backwards compatibility, does it?

Reply:
Nope, different signature